docs/docusaurus/docs/guides/connecting_to_your_data/fluent/data_assets/how_to_organize_batches_in_a_file_based_data_asset.md

```diff
@@ -96,23 +96,21 @@ For this guide, we will use a previously defined Datasource named `"my_datasourc
 
 To retrieve this Datasource, we will supply the `get_datasource(...)` method of our Data Context with the name of the Datasource we wish to retrieve:
 
-```python title="Python code"
-my_datasource = context.get_datasource("my_datasource")
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_pandas_filesystem_datasource.py my_datasource"
 ```
 
 ### 3. Create a `batching_regex`
 
 In a file-based Data Asset, any file that matches a provided regular expression (the `batching_regex` parameter) will be included as a Batch in the Data Asset.  Therefore, to organize multiple files into Batches in a single Data Asset we must define a regular expression that will match one or more of our source data files.
```

Comment:
This block seems quite similar to docs/docusaurus/docs/components/connect_to_data/filesystem/_tip_filesystem_data_asset_if_batching_regex_matches_multiple_files.md - would we be able to reuse that here using doucusauras?

Reply:
@TrangPham It is not easy to do now, and I would propose that we consider it for later.  The problem is that every integration test is largely independent, without sharing utilities (we do it for significant methods, but not for lightweight generic ones, such as `get_datasource()` or define a `batching_regex`.  So instead of using factored out snippets we repeat them as needed for each integrated tests.  Often, they are similar, but not identical, because each integration test has different purposes.  Thanks.